### PR TITLE
Type-grid-css

### PR DESCRIPTION
**Pull request by MilesAger**
Also updated Navbar feature placements.

### CODE REVIEW

File: client/src/components/BattleView.jsx [hunk 1]
```diff
@@ -135,24 +135,6 @@ const BattleView = () => {
...
-        rivalCheck = true;
-      }
-    }
-    if (!playerCheck) {
-      handleResult('loss');
-    } else if (!rivalCheck) {
-      handleResult('win');
-    }
-  };
-
```
Comment (colinwilliams91) [inline client/src/components/BattleView.jsx:155]: ???????? Did you just WildCard YEET this function into the oblivion???

**Review by colinwilliams91 — APPROVED** [review]
what happened to matchResolution() inside BattleView? We were using that function.

**Review by Benthusiast — APPROVED** [review]
deleted Battleview from this request and approved.

**Comment by Benthusiast** [comment]
WELP I GUESS THAT DELETES IT FROM THE REPO THATS INTUITIVE

**Pull request merged**